logging_utilities/filters/type_validator.py

```diff
@@ -0,0 +1,58 @@
+import logging
+
+
+def is_instance(attr, of_type):
+    if isinstance(of_type, str):
+        attr_class = type(attr)
+        if '.' in of_type:
+            return attr_class.__module__ + '.' + attr_class.__name__ == of_type or of_type in [
```

Comment:
Here to make the code clearer I would not use `+` but `format()`
```
'{}.{}'.format(attr_class.__module__, attr_class.__name__) == of_ype
```

Reply:
Would an f string also work?

Reply:
Oh I see f strings are probably not ok because they are only supported on python 3.6 and above.